slasher/utils.go

```diff
@@ -0,0 +1,25 @@
+// Package slasher implements slashing validation
```

Comment:
I meant under /shared/sliceutil, it seems general enough that should be shared across codebase

Reply:
great
moving it 
Thanks